projects/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -468,6 +468,16 @@ export class DatatableComponent implements OnInit, DoCheck, AfterViewInit, After
    */
   @Input() enableClearingSortState = false;
 
+  /**
+   * Throttle time in ms. Will recalculate table this time after the resize.
+   */
+  @Input() resizeThrottle = 100;
+
+  /**
+   * Emits first visibility change immediately without waiting for resizeThrottle.
+   */
+  @Input() emitInitial = true;
+
```

Comment:
@fh1ch currently we have `siResizeObserver` as part of our internal Element lib which is being used with datatable by consumers manually to trigger recalculate (people often forgets to do this and end up with issues). Goal of the MR is to remove that manual need however since `siResizeObserver` also allows these two props I am exposing the same here to keep it compatible with what we have today with `siResizeObserver`.